### PR TITLE
[APM-CI] bump Elastic Stack version to 7.x branch

### PR DESCRIPTION
**Pull request by kuisathaverat**
set the Elastic stack version to  7.0.0-beta1

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -23,7 +23,7 @@ pipeline {
     durabilityHint('PERFORMANCE_OPTIMIZED')
   }
   parameters {
-    string(name: 'ELASTIC_STACK_VERSION', defaultValue: "6.5", description: "Elastic Stack Git branch/tag to use")
+    string(name: 'ELASTIC_STACK_VERSION', defaultValue: "7.0.0-beta1", description: "Elastic Stack Git branch/tag to use")
```
Comment (simitt) [inline Jenkinsfile]: Afaic since this is not released yet you need to point to the correct docker images.

**Comment by kuisathaverat** [comment]
I see,  it should be 7.x

```
manifest for docker.elastic.co/kibana/kibana:7.0.0-beta1-SNAPSHOT not found
manifest for docker.elastic.co/apm/apm-server:7.0.0-beta1-SNAPSHOT not found
manifest for docker.elastic.co/elasticsearch/elasticsearch:7.0.0-beta1-SNAPSHOT not found
```

**Comment by graphaelli** [comment]
7.x snapshots aren't ready yet so let's go with `7.0.0`

**Comment by graphaelli** [comment]
how does this work / which images are used?  There are no 7.x images yet, right?

**Pull request merged**